tests/test_dbclient.py

```diff
@@ -0,0 +1,46 @@
+import os
+import tests.context
+import psycopg2
+import pytest
+import pandas as pd
+from core.db_client import DBClient
+
+@pytest.fixture
+def db_client():
+	"""
+	Maintain instance of DB Client
+	"""
+	return DBClient(config_filepath='tests/artifacts/db_client/database_config.json')
+
+def check_empty_category_labels(db_client):
```

Comment:
I'm not sure if I'm missing something (not sure if there's 1 database for each data provider or 1 for all), but when would the table be empty other than the moment when we have 0 active users?